css/creative.css

```diff
@@ -2278,13 +2278,24 @@ footer .social-media-icon:hover {
   display: flex;
   justify-content: space-between;
 }
+@media (max-width: 767px) {
+.tab-vertical.active {
+  background: linear-gradient(to bottom right, #f17013, #ffc400);
+  color: white;
+  font-size: 1.4rem;
+  transition: all 0.2s ease-in-out;
+  transform: scale(1);
+}
+}
```

Comment:
Indent lines 2282 to 2297.

Reply:
going to leave it for a new ticket, I'm thinking of changing the code a bit more to add the +/-.